l10n/en/firefox/new/download.ftl

```diff
@@ -0,0 +1,61 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/new/
+
+firefox-new-download-firefox = Download { -brand-name-firefox }
```

Comment:
I considered this yeah, and in the old system I would have said yes. From my understanding after talking with Peiying, Pontoon now offers a "string matching" feature, so it will suggest to copy over translations from other places where they already exist. As such, global shared files are not quite as useful as they used to be. We do still have some, but they are confined to individual components such as navigation, footer, download button etc. I think these make sense as they are grouped in a common way. For general purpose strings, we're trying to steer away from creating another main.lang equivalent (which grew unwieldly and hard to manage over time).